tests/docker/config/storage.conf

```diff
@@ -8,21 +8,14 @@ app {
     interface = "0.0.0.0"
   }
 
-  subject {
-    anonymous = false
-    realm = "internal"
-    name = "service-account-delta"
-  }
-
   storage {
     root-volume = "/tmp"
     protected-directory = "protected"
     fixer-enabled = false
   }
 
-  delta {
-    public-iri = "https://test.nexus.bbp.epfl.ch"
-    internal-iri = "http://delta:8080"
+  authorization {
```

Comment:
We can't really test this part in the integration tests anymore as we need the public keys  for the realm.
In the different envs, we will get notified if Delta can't interact with the storage though